packages/web/utils/__tests__/compare.spec.ts

```diff
@@ -52,3 +57,41 @@ cases(
     },
   }
 );
+
+describe("compareDefinedMember", () => {
+  it("should return -1 if member is in a but not in b", () => {
+    const a = { member: "value" };
+    const b = {};
+    expect(compareDefinedMember(a, b, "member")).toBe(-1);
+  });
+
+  it("should return 1 if member is in b but not in a", () => {
+    const a = {};
+    const b = { member: "value" };
+    expect(compareDefinedMember(a, b, "member")).toBe(1);
+  });
+
+  it("should return 0 if member is in both a and b", () => {
+    const a = { member: "value" };
+    const b = { member: "value" };
+    expect(compareDefinedMember(a, b, "member")).toBe(0);
+  });
+
+  it("should return 0 if member is in neither a nor b", () => {
+    const a = {};
+    const b = {};
+    expect(compareDefinedMember(a, b, "member")).toBe(0);
+  });
+
+  it("should return -1 if a is a value and b is a nil value", () => {
+    const a = { member: "value" };
+    const b = { member: undefined };
+    expect(compareDefinedMember(a, b, "member")).toBe(-1);
+  });
+
+  it("should return 0 if a and b are nil values", () => {
+    const a = { member: null };
+    const b = { member: undefined };
+    expect(compareDefinedMember(a, b, "member")).toBe(0);
+  });
+});
```

Comment:
Thoughts on using jest-in-case?

Reply:
My only opposition to it is it breaks my Jest UI extension for each of the test cases, however it does make tests easier to add to and update, so I'll make the change.